keyboards/chromatoneminipico/keymaps/led/rules.mk

```
@@ -0,0 +1,7 @@
+RGB_MATRIX_ENABLE = yes       # Use RGB matrix (Don't enable this when RGBLIGHT_ENABLE is used.)
+RGB_MATRIX_DRIVER = WS2812    # 2021/01/23 added.
```

Should fix build error
```suggestion
RGB_MATRIX_DRIVER = ws2812    # 2021/01/23 added.
```